ddclient.in

```diff
@@ -2146,6 +2154,128 @@ sub geturl {
     return $reply;
 }
 
+######################################################################
+## geturl -- via curl
+######################################################################
+sub curl_cmd {
+    my $params = join("\n",@_);
```

Comment:
This defeats the purpose of passing the args as a list. (Well, almost: instead of worrying about shell metacharacters we only have to worry about curl config file metacharacters.)

Reply:
I removed this and updated the `print` function to take the array directly.